src/main/java/ttt/CommandLineGameController.java

```diff
@@ -15,10 +17,13 @@
 import static ttt.ReplayOption.Y;
 
 public class CommandLineGameController {
+    private final GameConfiguration gameConfiguration;
     private GameRules gameRules;
     private Prompt gamePrompt;
+    private GameType gameType;
 
-    public CommandLineGameController(GameRules gameRules, Prompt commandLinePrompt) {
+    public CommandLineGameController(GameConfiguration gameConfiguration, GameRules gameRules, Prompt commandLinePrompt) {
```

Comment:
There is now some tension with the `GameConfiguration` and the `GameRules`. If you think about it, it looks like it would make sense to have the `GameRules` in the `GameConfiguration` (_i.e. you configure a game with the rules you want to apply to it_).

This does not mean that it's a _hard_ tension and should be changed necessarily, but it is some tension that requires us to think about the problem at hand and how we could make this tension go away.